config/rbac/vg_manager_role.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: vg-manager-role
+rules:
+- apiGroups:
+  - lvm.topolvm.io
+  - apps
+  resources:
+  - lvmclusters
+  - daemonsets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - lvm.topolvm.io
+  resources:
+  - lvmclusters/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - lvm.topolvm.io
+  resources:
+  - lvmclusters/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - privileged
```

Comment:
is this because the same problem that the one we have in topolvm node daemonset? Why do we use privileged SCC?

Reply:
yes. Fixed it in a last commit. Adding new SCC for vgmanager.

Reply:
Please include the scc changes in this commit.

Reply:
done.